status/flowcells.py

```diff
@@ -244,7 +255,13 @@ class FlowcellLinksDataHandler(SafeHandler):
     def get(self, flowcell):
         self.set_header("Content-type", "application/json")
         try:
-            p = lims.get_containers(name=flowcell[8:])[0]
```

Comment:
I understand your point... on the other hand a clear function name should avoid the need to look for what the function really does....